runtime/composable/src/lib.rs

```diff
@@ -850,6 +851,14 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl lending_runtime_api::LendingRuntimeApi<Block, AccountId, Balance, MarketId> for Runtime {
+		fn get_borrow_limit(_: MarketId, _: AccountId) -> SafeRpcWrapper<Balance> {
+
+			unimplemented!()
```

Comment:
Never ever introduce a `unimplemented` or `todo` or `unreachable` in any runtime if they can be reached. If this runtime doesn't contains the lending pallet we don't implement the RPC API for it.

Reply:
@benluelo ill wait for your refactor and then remove this

Reply:
i think it will not pass linters. so we can ignore that on review as it will not pass CI